tests/macros/test-draft.js

```diff
@@ -14,89 +14,89 @@ describeMacro('draft', function () {
     itMacro('No arguments (en-US)', function (macro) {
         return assert.eventually.equal(
             macro.call(),
-            `<div class="overheadIndicator draft">\n    <p><strong>Draft</strong><br/>\n    This page is not complete.</p>\n    \n</div>`
+            `<div class="blockIndicator draft">\n    <p><strong>Draft</strong><br/>\n    This page is not complete.</p>\n    \n</div>`
```

Comment:
No pages found using this macro (the name of the file also suggest it was temporary):
https://developer.mozilla.org/en-US/search?topic=none&kumascript_macros=test-draft&locale=*

Reply:
🤦🏻‍♂️ This is a `mocha` unit test file for ensuring that the `{{Draft}}` macro creates correct output.

Reply:
Doh! :) Right you are.